src/components/evaluation/EvaluationActions.vue

```diff
@@ -153,13 +168,15 @@ export default {
     afterApply(action) {
       /* TODO: a more informative screen reader alert plus a visual indication of the affected row(s) */
       this.isDuplicating = false
+      this.isloading = false
```

Comment:
@cesarvh I think we will need `this.isLoading = false` inside the `finally` block on line 199 instead of here - that way it will happen whether the API call succeeds or fails.

Reply:
Ah yes, that makes sense. Fixed